pkgs/misc/tmux-plugins/tmux-fingers/fix.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/tmux-fingers.tmux b/tmux-fingers.tmux
+index f15b509..847e651 100755
+--- a/tmux-fingers.tmux
++++ b/tmux-fingers.tmux
+@@ -2,11 +2,7 @@
+ 
+ CURRENT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ 
+-if command -v "tmux-fingers" &>/dev/null; then
+-  FINGERS_BINARY="tmux-fingers"
+-elif [[ -f "$CURRENT_DIR/bin/tmux-fingers" ]]; then
+-  FINGERS_BINARY="$CURRENT_DIR/bin/tmux-fingers"
+-fi
++FINGERS_BINARY="@tmuxFingersDir@/tmux-fingers"
```

Comment:
Since you're patching on the tmux script, I believe you could simplify it by trimming the whole file down to just `tmux run ${fingers}/bin/tmux-fingers load-config`. 
The install wizard isn’t really necessary for nix users anyway.